cms/templates/settings.html

```diff
@@ -199,10 +199,30 @@ <h3 class="title">${_("These Dates Are Not Used When Promoting Your Course")}</h
           </div>
           % endif
         </section>
+
+        <section class="group-settings schedule">
+          <header>
+            <h2 class="title-2">${_('Course Details')}</h2>
+            <span class="tip">${_('')}</span>
+          </header>
+          <ol class="list-input">
+            <li class="field" id="field-course-language">
+              <label for="course-language">${_("Course Language")}</label>
+              <select id="course-language">
+                <option disabled selected></option>
+                % for lang, label in language_options:
+                  <option value="${lang}">${label}</option>
+                % endfor
+              </select>
+              <span class="tip tip-stacked">${_('')}</span>
+            </li>
+          </ol>
+        </section>
+
         <hr class="divide" />
             <section class="group-settings marketing">
               <header>
-                <h2 class="title-2">${_("Introducing Your Course")}</h2>
+                <h2 class="title-2">${_("Course Introduction Page")}</h2>
```

Comment:
This seems like an odd change, that seems irrelevant to these changes - is there a reason for this now? (e.g. there's something already in master that changes it, and this avoids some other problem)